packages/mock/src/mocks/valueset.ts

```diff
@@ -8,8 +8,18 @@ export const exampleValueSet: ValueSet = {
     contains: [
       {
         system: 'x',
-        code: 'test-code',
-        display: 'Test Display',
+        code: 'test-code-1',
```

Comment:
I *think* this is fine, but there's minor risk that a random customer test is relying on this `code` or `display`.  If we wanted to be really safe, we'd leave these as-is, and add the new codes.

I'm probably fine with how you've done it though.

Reply:
Ah, good point. No reason for the churn. I'll switch this one back.